pkg/integrations/commands/list-resource-types.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+)
+
+func ListResourceTypes() *cobra.Command {
+	var integrationID string
+	cmd := &cobra.Command{
+		Use:     "resource-types",
+		GroupID: Group.ID,
+		Short:   "ListIntegrations all resource types of integration",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := aponoapi.GetClient(cmd.Context())
+			if err != nil {
+				return err
+			}
+
+			resp, _, err := client.AccessRequestsApi.GetSelectableResourceTypes(cmd.Context(), integrationID).Execute()
+			if err != nil {
+				return err
+			}
+
+			table := uitable.New()
+			table.AddRow("ID", "NAME")
```

Comment:
not sure if there is a resource-type id

Reply:
I might change the name to be the human readable label